portfolio/src/main/webapp/script.js

```diff
@@ -26,7 +26,11 @@ function promptRandomQuote() {
   // Pick a random quote.
     const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
-  // Show message.
-  // TO DO: make message a pop up
-    alert(quote)
+  // Return quote as string to be used elsewhere.
+    return quote;
 }
+
+function popupMessage() {
+    var quoteToDisplay = pickRandomQuote();
+    document.getElementById("quoteDisplayer").innerHTML = quoteToDisplay;
```

Comment:
Per [style guide](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes):

> Ordinary string literals are delimited with single quotes ('), rather than double quotes (").